awx/main/utils/common.py

```diff
@@ -1059,6 +1059,21 @@ def get_external_account(user):
     return account_type
 
 
+def get_existing_yaml_files(dirname, filename):
+    '''
+    Check either filename.yml or filename.yaml exists.
+    Return path of existing files as an array.
+    Return an empty array if neither exists.
+    '''
+    result = []
+    path_prefix = os.path.join(dirname, filename)
+    for ext in ['.yml', '.yaml']:
+        path = path_prefix + ext
+        if os.path.exists(path):
+            result.append(path)
+    return result
+
```

Comment:
@kandeh is there a reason you added this function that I might be missing?
Based on what I'm seeing in #4540, the functionality desired can be achieved without writing a function to do this.
I'm thinking an alternative would be to use os.path.isfile() to check for .yml vs. .yaml, since that library is already imported in tasks.py and will alleviate the need for this function.

Reply:
I considered it is possible to have the same situation in somewhere else in the future.
I just removed this function.